source/grid/grid_in.cc

```diff
@@ -835,7 +835,33 @@ GridIn<dim, spacedim>::read_unv(std::istream &in)
             n_entities;
 
           AssertThrow(in.fail() == false, ExcIO());
-          in >> id;
+          // Ocassionally we encounter IDs which are not integers - we don't
+          // support that case since there is no sane way for us to determine
+          // integer IDs from, e.g., strings.
```

Comment:
```suggestion
          // Occasionally we encounter IDs that are not integers - we don't
          // support that case since there is no sane way for us to determine
          // integer IDs from, e.g., strings.
```